jaxdecomp/fft.py

```diff
@@ -24,45 +26,36 @@ def _fft_norm(s: Array, func_name: str, norm: str) -> Array:
   raise ValueError(f'Invalid norm value {norm}; should be "backward",'
                    '"ortho" or "forward".')
 
-
+# Has to be jitted here because _fft_norm will act on non fully addressable global array
+# Which means this should be jit wrapped
+@partial(jit, static_argnums=(0,1,3))
```

Comment:
sorry, but why does this need to be jitted? _fft_norm is a trivial function that just needs to know the global size of the array, no distribution involved.

If the upstream jax code for the FFT is not jitted then maybe we don't need to jit here either.

Reply:
@ASKabalan , same question here